### ISSUE TITLE
Cannot read property children of null

### ISSUE THREAD

**Issue by black-snow** [johannhof/xpath-dom#1]
In `index.js`:42+ / `exports.getXPath()` I had to change:

``` javascript
    var siblings = node.parentNode.children;
```

to 

``` javascript
    if (node.parentNode === undefined || node.parentNode === null)
      break;

    var siblings = node.parentNode.children;
```

for `node.parentNode` was often null.

**Comment by black-snow** [comment]
PR here: https://github.com/johannhof/xpath-dom/pull/2

**Comment by johannhof** [comment]
Fixed by #2
